xml/hwbp_registers.xml

```diff
@@ -95,11 +95,23 @@
         <field name="data" bits="XLEN-1:0" access="R/W" reset="Preset" />
     </register>
 
-    <register name="Trigger Data 3" short="tdata3" address="0x7a3">
-        Trigger-specific data.
+    <register name="Trigger Extra" short="textra" address="0x7a3">
+        \begin{commentary}
+            ASIDLEN is limited in size when XLEN is less than DXLEN to preserve
+            the maximum number of bits for future use.
+        \end{commentary}
+        <field name="0" bits="XLEN-1:2+ASIDLEN" access="R" reset="0" />
+        <field name="tasid" bits="1+ASIDLEN:1" access="R/W" reset="0">
+            If XLEN is 32 then ASIDLEN is 9. Otherwise ASIDLEN is 16.
```

Comment:
Shouldn't it be as the priv spec defines it:

```
The number of ASID bits is also implementation-defined and may be zero.  The
number of implemented ASID bits, termed {\mbox {\em ASIDLEN}}, may be
determined by writing one to every bit position in the ASID field, then
reading back the value in {\tt satp} to see which bit positions in the ASID
field hold a one.  The least-significant bits of ASID are implemented first:
that is, if ASIDLEN~$>$~0, ASID[ASIDLEN-1:0] is writable.  The maximal value
of ASIDLEN, termed ASIDMAX, is 9 for Sv32 or 16 for Sv39 and Sv48.
```

Reply:
Then we don't need to redefine it at all here.

Reply:
I missed that language. Let me take that into account.